PROVESFlightControllerReference/Components/ProvesRouter/docs/sdd.md

```
@@ -0,0 +1,59 @@
+| `sync input` | `fileBufferReturnIn` | `Fw.BufferSend` | Receiving back ownership of buffer sent on `fileOut` and `unknownDataOut` |
+| `sync input` | `cmdResponseIn` | `Fw.CmdResponse` | Port for receiving command responses from a command dispatcher (can be a no-op) |
+| `output` | `unknownDataOut` | `Svc.ComDataWithContext` | Port forwarding unknown data (useful for adding custom routing rules with a project-defined router) |
+| `output` | `bufferAllocate` | `Fw.BufferGet` | Port for allocating buffers, allowing copy of received data |
+| `output` | `bufferDeallocate` | `Fw.BufferSend` | Port for deallocating buffers |
+| `output` | `packetRouted` | `Fw.Signal` | Emitted after each received packet is processed; used to reset command loss timer in ModeManager |
+
+## Requirements
+
+| Name | Description | Rationale | Validation |
+| ---- | ----------- | --------- | ---------- |
+SVC-ROUTER-001 | `Svc::ProvesRouter` shall route packets based on their packet type as read from the `ComCfg::FrameContext` APID field (`context.get_apid()`) | Routing mechanism of the F´ comms protocol | Integration test |
```

markdownlint-cli2 (0.23.0)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: trailing_only; Missing leading pipe

(MD055, table-pipe-style)